### PR TITLE
Say hello to Jekyll's New Lead Developer

### PR DESCRIPTION
**Pull request by parkr**
:tada: Welcome @oe as our new lead developer! :tada:

### CODE REVIEW

File: docs/team/index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+## Plugin Core
+
+*The Jekyll Plugin Core Team's responsibility is to ensure the development and
+community around the core plugins thrive. They also provide guidance in
+conversations about extensibility of Core Jekyll.*
+
+1. Ashwin (@ashmaroli)
+2. Florian (@Crunch09)
+3. Mert (@mertkahyaoglu, jekyll-admin)
+4. Chase (@Chase
```
Comment (DirtyF) [inline docs/team/index.md]: closing `)` missing

**Comment by oe** [comment]
@jekyllbot: merge +docs

File: docs/team/index.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ conversations about extensibility of Core Jekyll.*
 1. Ashwin (@ashmaroli)
 2. Florian (@Crunch09)
 3. Mert (@mertkahyaoglu, jekyll-admin)
-4. Chase (@Chase
+4. Alfred Xing (@alfredxing)
```
Comment (DirtyF) [inline docs/team/index.md]: @Ch4s3 is still maintaining `classifier-reborn` whether @alfredxing is not maintaining any plugin at the moment.

Comment (ashmaroli) [reply docs/team/index.md]: I'm not yet maintaining any of the *Official Plugins* as well..

Comment (Ch4s3) [reply docs/team/index.md]: Ahh, yes, I am still maintaining `classifier-reborn`.

**Pull request merged**